src/modules/drive/Toolbar/components/AddButton.jsx

```diff
@@ -25,7 +26,8 @@ export const AddButton = ({ className }) => {
         onClick={handleToggle}
         disabled={isDisabled || isOffline}
         startIcon={<Icon icon={PlusIcon} />}
```

Comment:
if it's this button 
![image](https://github.com/user-attachments/assets/74a9cbfa-ea7a-451b-8d8b-0f5104f7ef07)

I think you have to add `u-bdrs-xxx` to override the border-radius, and `u-o-50` or something near on the startIcon